src/third-party/arduino-json-5.2/include/ArduinoJson/Arduino/String.hpp

```diff
@@ -9,11 +9,8 @@
 
 #ifndef ARDUINO
 
-#ifndef ARDUINO_STRING_OVERRIDE

```

Comment:
restored the patch, and also added a `.patch` so we can easily apply it on every new version.

Best would be to submit a PR for it on https://github.com/bblanchon/ArduinoJson/